### PR TITLE
fix: test to check if country picker is visible was recreated

### PR DESCRIPTION
**Pull request by eimolina**
I recreated a test that had problems. I extracted the same code that uses the "PhoneInput" component, to have a component with the TestID field and be able to use it in the test.
Ref https://github.com/GaloyMoney/galoy-mobile/issues/430

### CODE REVIEW

**Comment by eimolina** [comment]
hi, @daviroo. Can you review this PR and if there are some changes to do, please let me know. tanks.

File: __tests__/screens/phone-auth.spec.tsx [hunk 1]
```diff
@@ -55,4 +54,25 @@ describe("WelcomePhoneInputScreen", () => {
       queryByPlaceholderText(translate("WelcomePhoneInputScreen.placeholder")),
     ).not.toBeNull()
   })
+  it("country picker is visible on press", async () => {
+    const { UNSAFE_queryByType, getByTestId } = render(
```
Comment (samerbuna) [inline __tests__/screens/phone-auth.spec.tsx:58]: `UNSAFE_queryByType` is not used

File: app/screens/phone-auth-screen/phone-auth.tsx [hunk 2]
```diff
@@ -256,6 +257,18 @@ export const WelcomePhoneInputScreen: ScreenType = ({
   } else {
     captchaContent = null
   }
+  const dropDown =
+    "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAi0lEQVRYR+3WuQ6AIBRE0eHL1T83FBqU5S1szdiY2NyTKcCAzU/Y3AcBXIALcIF0gRPAsehgugDEXnYQrUC88RIgfpuJ+MRrgFmILN4CjEYU4xJgFKIa1wB6Ec24FuBFiHELwIpQxa0ALUId9wAkhCnuBdQQ5ngP4I9wxXsBDyJ9m+8y/g9wAS7ABW4giBshQZji3AAAAABJRU5ErkJggg=="
```
Comment (samerbuna) [inline app/screens/phone-auth-screen/phone-auth.tsx:261]: Can we make this into an SVG?

**Review by samerbuna — CHANGES_REQUESTED** [review]

File: app/screens/phone-auth-screen/phone-auth.tsx (hunk 2 above)
Comment (eimolina) [reply app/screens/phone-auth-screen/phone-auth.tsx:261]: Yes, we can.

File: __tests__/screens/phone-auth.spec.tsx (hunk 1 above)
Comment (eimolina) [reply __tests__/screens/phone-auth.spec.tsx:58]: sorry, my bad.

**Review by samerbuna — APPROVED** [review]
Thank you!

**Pull request merged**